modules/apps/29-fee/keeper/escrow.go

```diff
@@ -214,21 +215,16 @@ func (k Keeper) RefundFeesOnChannelClosure(ctx sdk.Context, portID, channelID st
 				return err
```

Comment:
i think this error also still needs to be handled so improperly formatted refund addr doesnt stop channel closure?

Reply:
ah that's a good point, it's effectively the same situation. Nice catch!

Reply:
🤝